### PR TITLE
Activate on workspaces with `jest.config.{mjs|cjs|ts}`

### PR DESCRIPTION
**Pull request by felipecrs**
Like https://github.com/facebook/docusaurus

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,6 +32,7 @@
   ],
   "activationEvents": [
     "workspaceContains:jest.config.js",
+    "workspaceContains:jest.config.mjs",
     "workspaceContains:jest.json",
     "workspaceContains:node_modules/.bin/jest",
     "workspaceContains:node_modules/react-scripts/node_modules/.bin/jest",
```
Comment (felipecrs) [inline package.json:38]: PS: I think vscode does not watch these folders. Perhaps because there is a blacklist, or something.

**Comment by Araxeus** [comment]
I think that for clarity this PR should be renamed to `Fix extension not activating if jest.config is .mjs`
also technically it could also be `.ts` too

**Comment by felipecrs** [comment]
> I think that for clarity this PR should be renamed to `Fix extension not activating if jest.config is .mjs` also technically it could also be `.ts` too

Added `.ts` and `.cjs`. Also fixed the PR title, I agree.

**Review by connectdotz — APPROVED** [review]
looks good, thanks @felipecrs

**Pull request merged**